jwt.js

```diff
@@ -56,6 +69,79 @@ function fastifyJWT (fastify, opts, next) {
     options = options || {}
     return JWT.decode(token, options)
   }
+
+  function replySign (payload, options, next) {
+    if (typeof options === 'function') {
+      next = options
+      options = {}
+    } // support no options
+    var reply = this
+    if (next === undefined) {
+      return new Promise(function (resolve, reject) {
+        reply.jwtSign(payload, options, function (err, val) {
+          err ? reject(err) : resolve(val)
+        })
+      })
+    }
+
+    if (!payload) {
+      return next(new Error('jwtSign requires a payload'))
+    }
+    steed.waterfall([
+      function getSecret (callback) {
+        secretCallback(null, null, callback)
```

Comment:
Why `null, null`?

Reply:
The first null is replacing `request`. This method does not have access to the `request` object so I just left it as null. The second parameter is supposed to be payload and I have included that in the next commit.

Reply:
Actually you can access the request object with `reply.request` (I know, it's weird :D)